functions/sheets.js

```diff
@@ -117,12 +127,18 @@ const ReadSpreadsheet = {
       name: 'spreadsheetId',
       type: 'String',
     },
+    {
+      name: 'offset',
+      type: 'Number',
+      docs: 'Skip the first `offset` rows of the spreadsheet.',
+      defaultValue: 0,
```

Comment:
This does change the existing behavior!
If this is not fine, I can definitely set `1` as the default value and adjust my code :)

Reply:
No, this is a better default, thanks! I'll set this in my scripts this year.